src/cloudant/index.py

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/env python
+"""
+_utils_
+
+General utilities
+
+
+"""
+import json
+
+from collections import Sequence
+
+ALL_ARGS = [
+    "descending",
+    "endkey",
+    "endkey_docid",
+    "group",
+    "group_level",
+    "include_docs",
+    "inclusive_end",
+    "key",
+    "limit",
+    "reduce",
+    "skip",
+    "stale",
+    "startkey",
+    "startkey_docid"
+]
+
+BOOLEAN_ARGS = [
+    'include_docs',
+    'inclusive_end',
+    'reduce',
+    'group',
+    'descending'
+]
+
+ARRAY_ARGS = [
+    'startkey',
+    'endkey'
+]
+STRING_ARGS = [
+    'key',
+]
+
+
+def python_to_couch(kwargs):
```

Comment:
In this case its taking a dict and returning a dict, probably should call it something other than kwargs if it isnt ** prefixed...